### PR TITLE
adds mentortickets, and prevents mentors from mentorhelping

### PR DESCRIPTION
**Pull request by Amelia0010**
### Intent of your Pull Request
image

Accessible through a verb in the mentor tab
#### Changelog

:cl:  
rscadd: Mentors now have a ticket log system thingamajigg
/:cl:

### CODE REVIEW

**Comment by thehighwayman** [comment]
it's ugly though

**Comment by Nickvr628** [comment]
Why can't mentors mentorhelp?

**Comment by Amelia0010** [comment]
because then there'd be issues with whose logs the responses would have to be added to :)

File: yogstation/code/controllers/subsystem/yogs.dm [hunk 1]
```diff
@@ -1,11 +1,13 @@
-SUBSYSTEM_DEF(YogFeatures)
+SUBSYSTEM_DEF(Yogs)
 	name = "Yog Features"
 	flags = SS_BACKGROUND
 	init_order = -101 //last subsystem to initialize, and first to shut down
 
-/datum/controller/subsystem/YogFeatures/Initialize()
+	var/list/mentortickets = list() //less of a ticket, and more just a log of everything someone has mhelped, and the responses
```
Comment (ThatLing) [inline yogstation/code/controllers/subsystem/yogs.dm]:
```suggestion
	var/list/mentortickets //less of a ticket, and more just a log of everything someone has mhelped, and the responses
```

File: yogstation/code/controllers/subsystem/yogs.dm [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
+
+/datum/controller/subsystem/Yogs/Initialize()
```
Comment (ThatLing) [inline yogstation/code/controllers/subsystem/yogs.dm:8]:
```suggestion
/datum/controller/subsystem/Yogs/Initialize()
    mentortickets = list()
```

**Comment by Ktlwjec** [comment]
> it's ugly though

like u

**Comment by Time-Green** [comment]
freshboi69 - >mentorhelp ("I have a question")
mentor1 -> freshboi69 ("Sure, what is it?")
freshboi69 - > mentor1 ("how do i vore monkeys?")
mentor1 - > mentorhelp ("i have a question")

**Comment by Amelia0010** [comment]
what

**Pull request merged**